src/forms/create_ticket.ts

```diff
@@ -171,7 +177,7 @@ class FormFields extends BaseFormFields {
     }
 
     isSystemField(field: Users.Fields.UserField): boolean {
-        return (SystemFields.includes(String(field.type)) || field.system_field_options);
+        return (SystemFields.includes(String(field.type)) || field.system_field_options) as boolean;
```

Comment:
TS reports:
```
Property 'system_field_options' does not exist on type 'UserField'. Did you mean 'custom_field_options'?ts(2551)
```

Reply:
I created a custom ZDField type to handle this case.  https://github.com/mattermost/mattermost-app-zendesk/pull/34/files#diff-317fa1c3e53f7eddff384b174597cc031681277978d4f4ce2e88f13c0834fa5fR29